src/components/__tests__/Stats-test.tsx

```diff
@@ -12,58 +12,113 @@ import YearlyDataTab from "../YearlyDataTab";
 import MonthlyDataTab from "../MonthlyDataTab";
 
 describe("Stats", () => {
-  let wrapper;
-  let productionLibrary1 = modifyLibrary(testLibrary1, { "name": "Production Library 1", "registry_stage": "production" });
-  let productionLibrary2 = modifyLibrary(productionLibrary1, { "name": "Production Library 2" });
-  beforeEach(() => {
-    wrapper = Enzyme.mount(<Stats libraries={[testLibrary1, testLibrary2, productionLibrary1, productionLibrary2]}/>);
-  });
-
-  it("renders a panel with a header and tabs", () => {
+  it("renders a panel with a header", () => {
+    let wrapper;
+    let productionLibrary1 = modifyLibrary(testLibrary1, {
+      name: "Production Library 1",
+      registry_stage: "production",
+    });
+    let productionLibrary2 = modifyLibrary(productionLibrary1, {
+      name: "Production Library 2",
+    });
+    wrapper = Enzyme.mount(
+      <Stats
+        libraries={[
+          testLibrary1,
+          testLibrary2,
+          productionLibrary1,
+          productionLibrary2,
+        ]}
+      />
+    );
     let panel = wrapper.find(Panel);
     expect(panel.length).to.equal(1);
     expect(panel.prop("id")).to.equal("stats");
     expect(panel.prop("headerText")).to.equal("Aggregate Data");
-    let tabs = panel.find(Tabs);
-    expect(tabs.length).to.equal(1);
-    expect(tabs.find(".tab-nav").at(0).text()).to.equal("List");
-    expect(tabs.find(".tab-nav").at(1).text()).to.equal("Charts");
-    expect(tabs.find(".tab-nav").at(2).text()).to.equal("Adobe Data");
-    expect(tabs.find(".tab-nav").at(3).text()).to.equal("Yearly Data");
-    expect(tabs.find(".tab-nav").at(4).text()).to.equal("Monthly Data");
   });
 
-  it("sorts a list of libraries by their status", () => {
-    let sorted = wrapper.instance().sortLibraries();
-    expect(sorted.production).to.eql([productionLibrary1, productionLibrary2]);
-    expect(sorted.testing).to.eql([testLibrary1]);
-    expect(sorted.cancelled).to.eql([testLibrary2]);
-  });
+  describe("Open", () => {
+    let wrapper;
+    let productionLibrary1 = modifyLibrary(testLibrary1, {
+      name: "Production Library 1",
+      registry_stage: "production",
+    });
+    let productionLibrary2 = modifyLibrary(productionLibrary1, {
+      name: "Production Library 2",
+    });
+    beforeEach(() => {
+      wrapper = Enzyme.mount(
+        <Stats
+          libraries={[
+            testLibrary1,
+            testLibrary2,
+            productionLibrary1,
+            productionLibrary2,
+          ]}
+        />
+      );
+      let panelToggle = wrapper.find("button.panel-heading");
```

Comment:
New tests are good. Can you just write a comment here or above line 40 that explains why you need to first manually click on the panel to open it, such as "By default, the content of a Panel component is hidden. We need to manually trigger an onClick to open it to test the contents." or similar.

Reply:
Added!